src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -945,8 +946,8 @@
     <ProjectReference Include="..\Framework\Microsoft.Build.Framework.csproj" />
     <ProjectReference Include="..\Utilities\Microsoft.Build.Utilities.csproj" />
     
-    <Content Include="$(NuGetPackageRoot)\netstandard.library\2.0.0\build\netstandard2.0\ref\netstandard.dll" CopyToOutputDirectory="PreserveNewest" LinkBase="ref" />
-    <Content Include="$(NuGetPackageRoot)\netstandard.library\2.0.0\build\netstandard2.0\ref\mscorlib.dll" CopyToOutputDirectory="PreserveNewest" LinkBase="ref" />
+    <Content Include="$(NuGetPackageRoot)\netstandard.library\2.0.0\build\netstandard2.0\ref\netstandard.dll" CopyToOutputDirectory="PreserveNewest" LinkBase="ref" Condition="'$(TargetFramework)' == 'netstandard2.0'" />
```

Comment:
@AndyGerlicher any idea why this Content item is overriding the Reference for mscorlib and being passed to the compiler? that is what is causing the compile failures @crummel was trying to fix.

Reply:
I might have found a workaround - I moved this to a separate after-build target, this is working locally for me.